### ISSUE TITLE
[Teaser] RTE description adds blank paragraph tags

### ISSUE THREAD

**Issue by juliaetate** [adobe/aem-core-wcm-components#1105]
## Bug Report

**Current Behavior**
When editing the description text for a teaser component, additional empty paragraph tags are being added to the node. These may be added between paragraphs or at the end. The only way to remove them is in CRX/DE, but editing the text again will add additional paragraph tags. This does not happen when I change the node name for the text or to a new RTE field on the component so I believe it is tied to the JS somehow.

**Expected behavior/code**
RTE should not add empty paragraph tags when other text is added.

**Environment**
- AEM version 6.5.4.0
- Core Components version 2.10.0
- JRE version 1.8.0_171-b11

**Additional context / Screenshots**
![Teaser Issue](https://user-images.githubusercontent.com/18601409/87709229-9136c280-c769-11ea-855b-3528b93cc513.gif)

**Comment by sagarkarani** [comment]
This is a valid issue and it takes toll on google pagespeed.

**Comment by satishkumarjadhav** [comment]
@vladbailescu  Could you please confirm if this bug has been prioritized and working on fix ?

**Comment by gabrielwalt** [comment]
We'll address this issue in the next Core Component release, which we'll try to release by end of Sept, early Oct.

**Comment by blair-robertson** [comment]
Hello,

I did some investigation into this and it seems to be caused by the custom Teaser JS - probably for enable/disabling the Description RTE based upon the checkbox “Get Description from linked page”

When I disabled the commenting out `teaser.js` in `/apps/core/wcm/components/teaser/v1/teaser/clientlibs/editor/js.txt` then the problem disappears.

HTH

**Comment by msagolj** [comment]
Hi @blair-robertson, thank you for looking into this. I think this is an issue that is purely RTE related (RTE itself is not part of core comp) and got resolved meanwhile in one of the service packs.  I tested it with an older version of core comp (2.13) but with 6.5 SP 7, and i was not able to reproduce the issue neither the way @juliaetate (change the description) or you (enable/disable checkbox) described it.

@blair-robertson what AEM Version / SP did you use for testing ?
